Classes/ExpressionLanguage/DocumentTypeFunctionProvider.php

```diff
@@ -27,6 +27,8 @@
 use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
 use TYPO3\CMS\Extbase\Object\ObjectManager;
 
+const TYPE_OBJECT = "object";
+
```

Comment:
From my point of view it doesn't make sense to work with a constant here, but hard-code the same value at other places (like in `Classes/Controller/Embedded3dViewerController.php`). Either make it completely configurable (e. g. in `ext_conf_template.txt`) or just use the string value (like we do for newspapers and their components).